ambry-router/src/test/java/com.github.ambry.router/MockServerLayout.java

```diff
@@ -45,9 +48,6 @@ public MockServerLayout(ClusterMap clusterMap) {
    */
   public synchronized MockServer getMockServer(String host, int port) {
     DataNodeId dataNodeId = clustermap.getDataNodeId(host, port);
-    if (!mockServers.containsKey(dataNodeId)) {
-      mockServers.put(dataNodeId, new MockServer());
```

Comment:
Yes, if this class is used incorrectly, then it will throw NPE and we will come to know that it is being used incorrectly :)